taskRequests/details/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="/images/index.ico" type="image/x-icon" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;400;700;800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="/taskRequests/details/style.css" />
+
+    <title>Task Requests | Real Dev Squad</title>
+    <script src="/helpers/loadENV.js"></script>
+    <script src="/taskRequests/util.js"></script>
+    <script src="/taskRequests/constants.js"></script>
+    <script src="/taskRequests/details/script.js" defer></script>
+  </head>
+  <body>
```

Comment:
Would recommend using semantic tags if possible

Reply:
Thank you for your suggestion to use semantic HTML tags. While I understand the benefits of using semantic tags for improved accessibility and maintainability, in this particular case, I have decided to stick with the current structure of using `<div>` elements for styling and layout consistency. I've taken into consideration the project's design and styling requirements, and at this time, I believe the current approach aligns better with our design goals. If any specific accessibility issues arise, I'm committed to addressing them promptly. Thank you for your understanding.